packages/signers/CHANGELOG.md

```diff
@@ -0,0 +1,168 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+# 0.2.0 (2023-11-03)
```

Comment:
we are cutting a new release for light account factory update and sepolia arb support update, as v0.3.0. Think we should merge this in afterwards. cc: @moldy530

Reply:
agree!